### ISSUE TITLE
Max CFI: Review and publish new R&S page

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#71240]
## Deliverable
We need to publish a new page on VA.gov that provides information about submitting a CFI when you already receive the maximum amount of benefits for tinnitus.

## Resources
- [CAIA recommended content](https://dvagov-my.sharepoint.com/:w:/g/personal/anthony_procik_va_gov/EU5r_8YmwgdBhqt4BZsbbS8BLGer2R5bHK6YFaxJG55Ntg?e=1clCpA) (Word)
- [Product team draft content](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/Shared%20Documents/Employee%20Experience/Max%20CFIs/Max%20CFI%20explainer%20copy.docx?d=w4a7727703c734085a1aebf56275cfa67&csf=1&web=1&e=d7l9mw) (Word)
- DEPRECATED: [Product design](https://www.sketch.com/s/4319cad9-d81b-4b8e-8339-d4150fbd5274/v/eGplyz/p/F9B08E09-E3C2-4DE5-96FA-CE84537E11D6/canvas?posX=-7677.31124374378&posY=-3559.034495362688&zoom=1.055382490158081) (Sketch)
- NEW and SHINY: [Figma](https://www.figma.com/file/ZGsQLRV5tUTh1QJPCqefap/Max-CFI-design-refinements?type=design&node-id=26-2817&mode=design)

## Tasks
- [x] IA: Determine new page placement on VA.gov
- [x] SME: Review draft content from product team
- [x] AP, KF: Draft content based on product team and SME review
- [x] AP: Share content with product team for review
- [ ] KF: Create new page in Drupal
- [ ] AP: Share page preview with product team
- [ ] CL: Copyedit and publish the new page

## Criteria
- [ ] The new page has been published

**Comment by aprocik1** [comment]
@katherine-fung, thanks for attending tomorrow's meeting with the employee experience team. These are the questions we discussed asking:

- Can the team confirm that we need to use the content in [this screen](https://www.sketch.com/s/4319cad9-d81b-4b8e-8339-d4150fbd5274/a/uuid/A16885D8-8A06-46A1-9189-FC54BB131BD6) for the new page?
- Can the team confirm that we don't need to recommend updates to the content in [this screen](https://www.sketch.com/s/4319cad9-d81b-4b8e-8339-d4150fbd5274/a/uuid/7459571F-073F-4C66-A355-6FB3EA445ACA)?
- Can we attend the team's stakeholder review meeting next week?

**Comment by katherine-fung** [comment]
Hi @aprocik1, 

I'm updating the ticket with insights from the kickoff. 

1. The product team is meeting with the SMEs tomorrow to present the content "What's a maximum... rating?" They'll give SMEs a few days to review, and then give us the SME-reviewed content. I've explained that once we deliver our edits, they'll need to get another signoff from SMEs. They have invited us to tomorrow's meeting, and will also record this meeting in case we can't attend. (I told them I can't attend, I'll be out tomorrow). 

Given the order of operations, the 12/13 deadline isn't relevant anymore. 

2. The product team hasn't decided which scenario to use. They may decide in January after testing. In the Sketch file, on the "platform team feedback" page, we should offer edits to both the B (accordions) and C scenarios (external page).
 
3. We'll hold off on drafting the Drupal page until the team confirms they're going with scenario C.

**Comment by aprocik1** [comment]
Update: I'm moving this to blocked until we receive confirmation that the SMEs have reviewed the product team's draft content.

**Comment by aprocik1** [comment]
Hi @carodew! I know we're waiting on the VBA SME's accuracy review before offering our recommendations in your [Word document](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/Shared%20Documents/Employee%20Experience/Max%20CFIs/Max%20CFI%20explainer%20copy.docx?d=w4a7727703c734085a1aebf56275cfa67&csf=1&web=1&e=3PgShI). Our content team is OOO from Monday 12/25 to Monday 1/1, so we'll likely get to this in the new year. Let us know if you need any support before then. Thanks!

**Comment by carodew** [comment]
Just wanted to provide an update that we're still waiting on feedback on the language from our SMEs. Will post here when we have it. 

Also I transferred our Sketch files over to Figma last week, I'll update the links here.

**Comment by aprocik1** [comment]
Hi @amylai-nava! Thanks for inviting Katherine and I to today's meeting with OPO SMEs. After reviewing your [draft content](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/Shared%20Documents/Employee%20Experience/Max%20CFIs/Max%20CFI%20explainer%20copy.docx?d=w4a7727703c734085a1aebf56275cfa67&csf=1&web=1&e=Geerfg), we have a couple follow up questions.

- Where will the content live - within the tool or on a new resources and support page? Kristin's [IA recommendation](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69856#issuecomment-1854810922) is to place it on a new resources and support page. That would allow us to add a "Related information" section with links to other helpful pages, like the disability ratings page. And we could add a "Need more help?" section with contact information.
- What additional content needs to be added? The OPO SMEs referenced other content related to max CFIs.
- When should we review the content to offer our recommendations? If possible, we'd like to review the content before the OPO SMEs re-review and before user research begins.

**Comment by ChristineRoseSteiffer** [comment]
Hi @aprocik1 - I'm responding on behalf of the EE team to your questions above, as well as including a request from our team at the end. 

To you first question, we are happy to go with Kristin's recommendation for creating a new resources and support page. To your second question, we have not specifically discussed what additional content should be included on this page. We can reach back out to the OPO SME's to see what they recommend.

In the meantime, we were wondering if CAIA could help us improve the language in our educational content around extra-schedular (XS) ratings based on the feedback from OPO.

Our current content includes this language:
_To qualify for an extra-schedular rating, you must show:

* The Rating Schedule does not adequately describe your symptoms or experience, and
* You have other evidence of an unusual disability experience, such as missed work or frequent hospitalizations._

We think the second bullet is too vague, and may even encourage people to think they could get an XS rating. We had hoped Comp Service could provide more specific criteria for XS, but they’re not able to because the rules just aren’t that clear-cut. There needs to be flexibility in the rules so that adjudicators can make nuanced decisions based on the specific medical circumstances of the individual Veteran.

We have a couple ideas about how to move forward, including:

* Getting rid of the second bullet altogether, since it’s too vague to be meaningful
* Strengthen some of the language to explain that the disability experience must be exceptional, i.e. much worse than the typical symptoms for the condition at the highest standard rating level.

Would CAIA be able to come up with some options for this text? We can try to understand what other content may be able to cohabitate on this page in the meantime, and let you know what we find. 

Let me know your thoughts on this - thank you!

**Comment by carodew** [comment]
Hello, I've been out sick, so thanks @ChristineRoseSteiffer for the very thorough response here. 

I just want to add a little bit more context. We are thinking this info primarily lives on a new page, however we're not sure how we want to provide this information to people who are in the process of filling out the 526EZ online. We are testing two versions -- one where they have a link to this external page that opens in a new tab/window, with the idea that they can close this tab to return to the form flow. However we're not sure how well this will work as we've heard that Veterans often struggle to get back into the form flow in similar scenarios. 

The other option has this language appearing within the 526EZ form itself, on the page where the Veteran is selecting which disabilities they'd like to claim an increase for. Even if we go with this option, I think we'll still have that info on the separate, external page, as it could be useful to Veterans who are not currently filling out an application. 

You can check out the Figma files (linked in the original issue above) to see the two examples.

**Comment by aprocik1** [comment]
@carodew and @ChristineRoseSteiffer, thanks for your response to our questions! Katherine and I took drafted content for the resources and support page, which can be repurposed for the in-form view, as needed.

Here's [our recommended content](https://dvagov-my.sharepoint.com/:w:/g/personal/anthony_procik_va_gov/EU5r_8YmwgdBhqt4BZsbbS8BLGer2R5bHK6YFaxJG55Ntg?e=1clCpA) - in a copy of the document you shared with OPO. We left a comment and question for you about the "how to apply" content.

Note: We know you're exploring whether you can expand the scope of this work beyond tinnitus. If you can, Katherine and I will want to work with OPO to better understand the full impact of an extraschedular rating. We may need to include content in our resources and support page about how that rating can impact other benefits, like special monthly compensation.

**Comment by ChristineRoseSteiffer** [comment]
Hi @aprocik1 - thanks for reaching out! We just finished our research sessions last week (delayed due to many issues with no shows and cancellations), so we're in the middle of synthesis and analysis right now. I can follow up with you by the end of this week to see if we have a better grasp on where to go from here.

**Comment by ChristineRoseSteiffer** [comment]
Hey @aprocik1 just an update on where we are at with this ticket. We are still writing up our research report, but we found in the tests that both prototypes encouraged the participants to file a claim for increase which is the opposite behavior we want to encourage. We want to expand the scope of our current content to options beyond extra-schedular ratings, which has the lowest chance of getting the Veteran more benefits, with content related to secondary conditions, ancillary benefits, and individual unemployability which all have a higher chance of providing benefits to the Veteran. In order to do this, we are setting up interviews with VSRs next week to help us determine what that content should be. We will be back in touch with your team once we have a better grasp on what that content might look like so that you can help us to refine it. 

Let me know if you have any questions or concerns. Thanks for all your support!

**Comment by ChristineRoseSteiffer** [comment]
Hey @aprocik1 - I want to provide you & the team an update on this ticket. We have completed our [research report](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/abd-vro/2024-01-Max-CFI-Refinements/2024-01-Max-CFI-Refinements-Report.md) from our last round of testing. Our primary recommendation is a shift in the focus of our educational content from extra-schedular to other benefits more likely to provide value to the Veteran. We are wondering how much your team would like to be involved in our research of these other benefits and the crafting of that original content? Our plan is to conduct research with VSR/RVSRs this/next week to understand more about secondary conditions, ancillary benefits and individual unemployability. Would you or anyone from your team like to join those conversations? The plan is to take what we learn from them and craft some different content for the accordions on the Rated Disabilities page, but we could also discuss the creation of a page elsewhere dedicated to this theme of getting additional benefits. 

We would be happy to talk through our report and these recommendations with anyone from your team in more detail. I'd also like to introduce you to my colleague @andreamurchie who has joined us on this project and will be taking the lead on helping to shape some of this content. Let me know if you'd like to schedule a quick call or if you just want to stay updated async. Thanks so much!

**Comment by aprocik1** [comment]
@ChristineRoseSteiffer, thanks for the update! Katherine and I don't have the capacity to join your research, but we can draft any additional content needed. If you share synthesized research findings, we can review them alongside the research report. And then we can draft content for the accordions on the Rated Disabilities screen (and potentially a separate additional benefits / extra-schedular webpage). Do you have a sense of when we could expect research findings from your team? And do you have sense of when you'll need us to draft the new content by?

@andreamurchie, thanks for joining the work!

**Comment by ChristineRoseSteiffer** [comment]
Hi @aprocik1 - we're writing up both our findings from RVSR research and the draft content this week. We'll plan to have it to your team for review by early next week. We would like to work with your team to get the content to a point that we can do an additional round of testing on it that is just focused on testing the content itself. If you have any thoughts on how to do that best, we'd love to hear them. So no fixed deadline necessarily on when we need it - I guess I would say as soon as it is easily feasible for your team. Thank you!

**Comment by ChristineRoseSteiffer** [comment]
Hi @aprocik1 - here is our update on this ticket.

The report for our RVSR research is now posted in [Sharepoint](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/Shared%20Documents/Employee%20Experience/Max%20CFIs/Background,%20SME,%20and%20RVSR%20research/Max%20CFI%20All%20Conditions%20-%20RVSR%20Research%20Report.docx?d=w7d364c668f9a4bb98cf140de0778a93c&csf=1&web=1&e=M80QsM) which we invite you to review. To summarize the results from our conversations with raters, we found that the alternative benefits we were initially considering detailing at the bottom of the 526 form are still relatively rare, therefore we think this information might be better described elsewhere. We would love to jump on a call with you and your team sometime this or next week to discuss that recommendation and how we might move forward. We had already started drafting what our potential revised content might look like, and have included that in the Mural below. The only difference is that instead of this content living inside of accordions on the 526 form flow, we think it should be housed elsewhere within the IA of the site. Can you let me know a good time for a more detailed conversation together sometime soon? Thanks! CC @andreamurchie 

![Max CFI_Max Rating Alternative Paths_Draft Content](https://github.com/department-of-veterans-affairs/va.gov-team/assets/127338055/c643301a-b64f-493a-8046-57ab13b005fd)

**Comment by aprocik1** [comment]
@ChristineRoseSteiffer, thanks for the update! I'll invite my content and IA colleagues to our meeting. Are you available Wednesday 3/27 or Wednesday 4/3 between 12-1 pm ET?

Our sitewide CAIA team reserves those times for kickoffs and check ins, so I know we're all available then.

**Comment by aprocik1** [comment]
Notes from our 3/27 sync:
- The content team will return to this work in the coming weeks
- The content team will explore the need for a new page on ancillary benefits
    - Consider crosslinking to existing VA.gov pages for these benefits
    - Consider adding content from the previous iteration of the max CFI page
    - Consider adding content or findings from RVSR research
    - Driving question for the new page: If I'm at the maximum rating for my disability and my condition has gotten worse, what ancillary benefits or other alternatives can I purse for additional compensation?
- The disability benefits employee experience team will share their Mural board with the content team

**Comment by ChristineRoseSteiffer** [comment]
Hi @aprocik1 @katherine-fung @strelichl @kristinoletmuskat - thanks again for meeting with us earlier this week to discuss updates for this content. Here is the [link](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1711557420101/8420c5868ed7cc9a02f53a027750a8f6167830da?sender=u4cef3aa06e6e500fdad48741) to the mural that has the draft content which you all should be able to access - please let me know if you have any issues. You'll want to focus on the section titled, 'Pathways to more compensation (Draft Content)'. 

Also here is a [link](https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/_layouts/15/Doc.aspx?sourcedoc=%7B30CEA9A4-3F8B-4C66-B1A8-FB73D7031404%7D&file=Max%20CFI%20Alternative%20Pathways%20-%20Desk%20research%20summary.docx&action=default&mobileredirect=true) to some additional desk research @KimLadin did to understand more about the specific additional benefits available to Veterans. This information could be helpful to your team when you are able to pick up the ticket back up.

Thanks again for all of your support with this! We look forward to working together again soon :) FYI @andreamurchie

**Comment by katherine-fung** [comment]
Hey @ChristineRoseSteiffer, did your team implement the draft content (V3) in the 526 application (per the mockup above from the Mural file)? 

Is there still a request to create an R&S page to let Veterans know about alternative benefits to get additional compensation if they have the maximum standard rating? 

cc @strelichl @aprocik1

**Comment by mistymg** [comment]
Reached out [in Slack](https://dsva.slack.com/archives/C03C8661PAP/p1747665199088239) to see if this ticket is still needed/relevant.

**Comment by mistymg** [comment]
I chatted with several members of this product team [in Slack](https://dsva.slack.com/archives/C03C8661PAP/p1747678086426979?thread_ts=1747665199.088239&cid=C03C8661PAP) about this old ticket. They confirmed that adding additional content in the Form 526 didn't test well so they never implemented. There may be a benefit eventually to having some of this explanatory information in the applicable hub or an R&S page. But it's not a priority and we can close this ticket out.
